docgen/src/getting-started.md

```diff
@@ -10,7 +10,7 @@ githubSource: docgen/src/getting-started.md
 ---
 
 *This guide will walk you through the few steps needed to start a project with InstantSearch iOS.
-We will start from an empty iOS project, and create from scratch a full search interface!*
+We will start from an empty iOS project, and then create from scratch a full search interface!*
```

Comment:
>and then create a full search interface 

sounds better IMO

Reply:
I quickly wanted to make sure that the fastlane deploy command actually works and deploys the website 🙂